dll/win32/shell32/lang/ro-RO.rc

```diff
@@ -786,21 +786,21 @@ BEGIN
     IDS_ABOUT_VERSION_STRING "Versiune %S (%S)"
 
     /* context menus */
-    IDS_VIEW_LARGE "D&ale"
+    IDS_VIEW_LARGE "&Dale"
     IDS_VIEW_SMALL "&Pictograme"
     IDS_VIEW_LIST "&Listă"
     IDS_VIEW_DETAILS "&Detalii"
     IDS_SELECT "Selectea&ză"
     IDS_OPEN "&Deschide"
     IDS_CREATELINK "Creează sc&urtătură"
-    IDS_COPY "&Copiază"
-    IDS_DELETE "Șt&erge"
+    IDS_COPY "&Copiere"
+    IDS_DELETE "Ş&tergere"
     IDS_PROPERTIES "&Proprietăți"
-    IDS_CUT "&Decupează"
+    IDS_CUT "&Decupare"
     IDS_RESTORE "&Restabilește"
     IDS_FORMATDRIVE "F&ormatare…"
-    IDS_RENAME "Rede&numește"
-    IDS_PASTE "&Lipește"
+    IDS_RENAME "Redenu&mire"
+    IDS_PASTE "&Lipire"
     IDS_EJECT "S&coate"
     IDS_DISCONNECT "Deconectea&ză"
     IDS_OPENFILELOCATION "&Deschide locația fișierului"
```

Comment:
Can you find an accel for openFileLocation in your MS resources as well? Just asking because you do use the &D for IDS_CUT now. Not sure, whether both of those controls can happen in the same submenu or not.